src/Api/GetRc.ts

```diff
@@ -60,12 +59,12 @@ const walkUpTree = (startDir: string, endDir: string, callback: (path: string) =
   }
 }
 
-export const getConfig = (dir: string = process.cwd()): ConfigShape => {
+export const getConfig = (dir: string = process.cwd()): Config => {
   if (configFileName === '') {
     throw new Error('What is your config file name?  Use setConfigName.')
   }
 
-  const configs: (ConfigShape | null)[] = []
+  const configs: (Config | null)[] = []
```

Comment:
let's remove this logic including combining configs, we should remove merging, our configs are not flat, but deep. So we support only the first it is found.